src/openassetio-core/include/openassetio/LoggerInterface.hpp

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2013-2022 The Foundry Visionmongers Ltd
+
+#include <openassetio/export.h>
+#include <openassetio/enum.hpp>
+#include <openassetio/typedefs.hpp>
+
+#pragma once
+namespace openassetio {
+inline namespace OPENASSETIO_CORE_ABI_VERSION {
+OPENASSETIO_DECLARE_PTR(LoggerInterface)
+/**
+ * An abstract base class that defines the receiving interface for
+ * log messages generated a @ref manager or the API middleware.
+ */
+class OPENASSETIO_CORE_EXPORT LoggerInterface {
+ public:
+  /**
+   * @name Log Severity
+   * @{
+   */
+  enum Severity : EnumIdx { kCritical = 0, kError, kWarning, kProgress, kInfo, kDebug, kDebugApi };
+
+  static constexpr EnumNames<7> kSeverityNames{"critical", "error", "warning", "progress",
+                                               "info",     "debug", "debugApi"};
```

Comment:
[question] Was the extra white space intentional?

Reply:
That's `clang-format` aligning things.  I don't particularly like it either.